x-pack/docs/en/security/fips-140-compliance.asciidoc

```diff
@@ -131,13 +131,26 @@ NOTE: You can still use one of the plain `pbkdf2` options instead of `pbkdf2_str
 you have external policies and tools that can ensure all user passwords for the reserved,
 native, and file realms are longer than 14 bytes.
 
-You must set the `cache.hash_algo` realm settings
-and the `xpack.security.authc.password_hashing.algorithm` setting to one of the
+You must set the `xpack.security.authc.password_hashing.algorithm` setting to one of the
 available `pbkdf_stretch_*` values.
 When FIPS-140 mode is enabled, the default value for
 `xpack.security.authc.password_hashing.algorithm` is `pbkdf2_stretch`.
 See <<hashing-settings>>.
 
+For cache hashing, salted `sha256` is recommended instead of `PBKDF2`. Though compliant
+with FIPS-140-2, `PBKDF2` is -- by design -- slow, and thus not suitable as a cache hashing
+algorithm. Cached credentials are never stored on disk, and salted `sha256` provides
+an adequate level of security for in-memory credential hashing, without imposing prohibitive
+performance overhead.
+
+You should set the `cache.hash_algo` realm settings to `ssha256`. `ssha256` is the default value
+for all `cache.hash_algo` realm settings.
+See <<hashing-settings>>.
+
+You _may_ set the `cache.hash_algo` realm settings to any of the `pbkdf2` options, however you
```

Comment:
Not sure we need this paragraph, and happy to remove it if we don't think it adds value. Essentially, `pbkdf2` is a bad idea but it's not technically wrong and if a customer decides they want to use it and that the performance hit is acceptable, they can, from a FIPS standpoint. Wdyt?

Reply:
I think this paragraph should go along with the previous one that says about `pbkdf2` not suitable for caching hashing.